runtime/wasm/src/module/mod.rs

```diff
@@ -93,7 +94,8 @@ where
     ) -> Result<Self, FailureError> {
         let logger = logger.new(o!("component" => "WasmiModule"));
 
-        let parsed_module = config.data_source.mapping.runtime;
+        // FIXME: Cloning this may be expensive.
+        let parsed_module = config.data_source.mapping.runtime.clone();
```

Comment:
Same note as above about putting this in an `Arc` so we can clone it.

Reply:
Done